global/log.h

```diff
@@ -0,0 +1,42 @@
+//=============================================================================
+//  MuseScore
+//  Music Composition & Notation
+//
+//  Copyright (C) 2020 MuseScore BVBA and others
+//
+//  This program is free software; you can redistribute it and/or modify
+//  it under the terms of the GNU General Public License version 2.
+//
+//  This program is distributed in the hope that it will be useful,
+//  but WITHOUT ANY WARRANTY; without even the implied warranty of
+//  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+//  GNU General Public License for more details.
+//
+//  You should have received a copy of the GNU General Public License
+//  along with this program; if not, write to the Free Software
+//  Foundation, Inc., 675 Mass Ave, Cambridge, MA 02139, USA.
+//=============================================================================
+
+#ifndef MS_LOG_H
+#define MS_LOG_H
+
+#include <QDebug>
+
+#define LOGD() qDebug()
+#define LOGI() qInfo()
+#define LOGW() qWarning()
+#define LOGE() qCritical()
+
+#define IF_ASSERT(cond) if (!(cond)) { \
```

Comment:
Would it perhaps be better to rename it to something like `IF_FAILED_ASSERT` or `IF_ASSERT_FAILED`? With the current name it sounds more like a usual `if` statement with the condition which is normally assumed to be always `true`, so the block after `IF_ASSERT` is perceived at first glance as being always executed (but in fact the opposite takes place).

Reply:
I agree, renamed `IF_ASSERT` to `IF_ASSERT_FAILED`